openerp/openupgrade/doc/source/modules70-80.rst

```diff
@@ -416,7 +416,7 @@ Status :
 +-----------------------------------+-----------------------------------+
 |share                              | Nothing to do                     |
 +-----------------------------------+-----------------------------------+
-|stock                              | Done (experimental)               |
+|stock                              | Done                              |
```

Comment:
Any experiences with multi-company + multi-warehouse? That was the aspect that I was unsure of when I added my bit to the stock migration script.